cli/src/chains_config/mod.rs

```diff
@@ -112,7 +112,34 @@ pub(crate) fn update_chains_config(chains_opts: ChainsOpts) -> Result<()> {
                     },
                 });
             }
-            None => bail!("No chain {} found!", chain.name),
+            None => {
+                eprintln!("No chain {} found in config template, using default values instead", chain.name);
+                chains.push(Chain {
+                    name: chain.name.clone(),
```

Comment:
The `name` for the metadata portal is different from the name that we have in `chains.json` config file.

The name should be fetched from the chain spec. 
The example is here https://github.com/nova-wallet/metadata-portal/blob/4bab32ada0ae6e4b215886972be14083a4aa476b/cli/src/fetch.rs#L39

Use `specs.name` https://github.com/nova-wallet/metadata-portal/blob/4bab32ada0ae6e4b215886972be14083a4aa476b/cli/src/fetch.rs#L50